responsibleai/tests/data_balance/test_raimitigations.py

```diff
@@ -0,0 +1,89 @@
+# Copyright (c) Microsoft Corporation
+# Licensed under the MIT License.
+
+import json
```

Comment:
The module does have its own unit tests, but I included these tests so that in the CI/CD pipeline, we can be confident of the behavior / expected dataframe schema even if the module version updates.

For example, I currently stated `raimitigations~=0.0.3` (which means whenever 0.0.4 comes out, that will fulfill this dependency), so I just wanted to make sure that we can be confident about the dataframe schema even with the version update.

What do you think? Keep or remove?